src/pages/RepoPage/BundlesTab/BundleContent/BundleSummary/hooks/useBundleBranchSelector.ts

```diff
@@ -0,0 +1,59 @@
+import { useParams } from 'react-router-dom'
+
+import { Branch, useBranch } from 'services/branches'
+
+interface URLParams {
+  provider: string
+  owner: string
+  repo: string
+  branch?: string
+  ref?: string
+}
+
+const getDecodedBranch = (branch?: string) =>
+  !!branch ? decodeURIComponent(branch) : branch
+
+const getDecodedRef = (ref?: string) => (!!ref ? decodeURIComponent(ref) : ref)
+
+interface UseBundleBranchSelectorProps {
+  branches: Branch[] | null
+  defaultBranch: string
+}
+
+export function useBundleBranchSelector({
```

Comment:
Is this essentially the same as src/pages/RepoPage/CoverageTab/hooks/useBranchSelector.js?

Reply:
Yes it is pretty much the same